### PR TITLE
fix: Add DiffSuppressFunc for firewall IPv6 fields

### LINKED ISSUE #634: [Bug]: uneeded changes in firewall with ipv6
### Terraform Version

Terraform v1.1.9 on linux_amd64

### Linode Provider Version

version = "1.27.1"

### Effected Terraform Resources

linode_firewall

### Terraform Config Files

```terraform
resource "linode_firewall" "test" {
  label = "test"
  tags  = []

  inbound_policy = "DROP"
  
  inbound {
    label    = "allow"
    action   = "ACCEPT"
    protocol = "TCP"
    ports    = "1234"
    ipv6     = [linode_instance.instance2.ipv6]
  }

  outbound_policy = "ACCEPT"

  linodes = [linode_instance.instance1.id]
}
```

### Debug Output

```
  ~ update in-place

Terraform will perform the following actions:

  # linode_firewall.test will be updated in-place
  ~ resource "linode_firewall" "test" {
        id              = "123"
        tags            = []
        # (7 unchanged attributes hidden)

      ~ inbound {
          ~ ipv6     = [
              - "1111:1111::1111:1111:1111:f88/128",
              + "1111:1111::1111:1111:1111:0f88/128",
            ]
            # (5 unchanged attributes hidden)
        }
    }
```

### Panic Output

_No response_

### Expected Behavior

No change in firewall

### Actual Behavior

For some ipv6 IPs terraform apply change the ip from 3 characters to 4 characters although the ip doesn't change:
1111:1111::1111:1111:1111:f88/128
1111:1111::1111:1111:1111:0f88/128

### Steps to Reproduce

terraform apply

### PR DESCRIPTION
**Pull request by LBGarber**
This change will suppress diffs on different but equivalent IPv6 ranges in the `linode_firewall` resource.

Resolves #634

### CODE REVIEW

File: linode/helper/normalize_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	result, err := helper.CompareIPv6Ranges(ips[0], ips[1])
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !result {
+		t.Fatalf("ranges are reported as different despite being equal")
+	}
+
+	ips[1] = "1111:1111::1111:1211:1111:0f88/126"
```
Comment (zirpu) [inline linode/helper/normalize_test.go:24]: shouldn't this be a /128?

Comment (LBGarber) [reply linode/helper/normalize_test.go:24]: For this part of the test I made a few arbitrary changes to test that comparing unequal IPv6 ranges will return false. It's probably a good idea to test mask and IP differences separately, so I'll throw in a fix for that.

**Review by jriddle-linode — APPROVED** [review]

**Pull request merged**